templates/taxi/car_detail.html

```diff
@@ -1,12 +1,19 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
-  <hr>
-  <ul>
-  {% for driver in car.drivers.all %}
-    <li>{{ driver.first_name }} {{ driver.last_name }}</li>
-  {% endfor %}
-  </ul>
+  <h1 class="taxi-title">{{ car.manufacturer.name }} {{ car.model }}</h1>
+  {% if car.drivers %}
```

Comment:
Ensure that `car.drivers` is a related manager (e.g., a ManyToManyField or ForeignKey with related_name) that supports the `.all()` method. If `drivers` is not a related manager, this could lead to an error.